test/action.spec.js

```diff
@@ -0,0 +1,112 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const mock = require('mock-require');
+const log = require('npmlog');
+const path = require('path');
+
+const action = (config, args) => require('../src/action')(config, args || []);
```

Comment:
Yes, but I am keeping the naming with current master. You can submit a PR but maybe let's wait till these two get merged and ideally #10 so that we are clean?